STM32Cube/.settings/com.st.stm32cube.ide.mcu.sfrview.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
likewise, .settings should be getting gitignored. I assume this is from a commit before the gitignore existed - pls delete

Reply:
the .gitignore was not capturing the settings folder as it wasn't prefixed with `**`